satpy/etc/composites/ahi.yaml

```
@@ -16,9 +16,50 @@ modifiers:
 
 composites:
   green:
+    # Deprecated green composite using deprecated class 'GreenCorrector'. Use 'hybrid_green' instead.
     compositor: !!python/name:satpy.composites.spectral.GreenCorrector
     # FUTURE: Set a wavelength...see what happens. Dependency finding
     #         probably wouldn't work.
+    prerequisites:
+      # should we be using the most corrected or least corrected inputs?
+      # what happens if something requests more modifiers on top of this?
+      - wavelength: 0.51
+        modifiers: [ sunz_corrected, rayleigh_corrected ]
+      - wavelength: 0.85
+        modifiers: [ sunz_corrected ]
+    standard_name: toa_bidirectional_reflectance
+
+  green_true_color_reproduction:
+    # Deprecated green composite using deprecated class 'GreenCorrector'. Use 'reproduced_green' instead.
+    # JMA True Color Reproduction green band
+    # http://www.jma.go.jp/jma/jma-eng/satellite/introduction/TCR.html
+    compositor: !!python/name:satpy.composites.spectral.GreenCorrector
+    fractions: [ 0.6321, 0.2928, 0.0751 ]
+    prerequisites:
+      - name: B02
+        modifiers: [ sunz_corrected, rayleigh_corrected ]
+      - name: B03
+        modifiers: [ sunz_corrected, rayleigh_corrected ]
+      - name: B04
+        modifiers: [ sunz_corrected ]
```

Other composites don't have spaces before/after first/last list items.
```suggestion
        modifiers: [sunz_corrected, rayleigh_corrected]
      - wavelength: 0.85
        modifiers:  sunz_corrected]
    standard_name: toa_bidirectional_reflectance

  green_true_color_reproduction:
    # Deprecated green composite using deprecated class 'GreenCorrector'. Use 'reproduced_green' instead.
    # JMA True Color Reproduction green band
    # http://www.jma.go.jp/jma/jma-eng/satellite/introduction/TCR.html
    compositor: !!python/name:satpy.composites.spectral.GreenCorrector
    fractions: [0.6321, 0.2928, 0.0751]
    prerequisites:
      - name: B02
        modifiers: [sunz_corrected, rayleigh_corrected]
      - name: B03
        modifiers: [sunz_corrected, rayleigh_corrected]
      - name: B04
        modifiers: [sunz_corrected]
```